app/components/participant-creator/component.js

```diff
@@ -73,6 +73,12 @@ export default Ember.Component.extend({
             });
             if (!useAsPassword) {
                 password = makeId(10);
+                this.get('extra').pushObject({
+                    key: 'password',
+                    value: password
+                });
+                extra['password'] = password;
+                this.set('useAsPassword', 'password');
```

Comment:
@abought, is it okay if I refactor how the password gets stored in a follow-up PR? That way, it does not need to block the rest of the changes in this PR.